unit_test/test_spdm_requester/get_measurement_extension_log.c

```diff
@@ -0,0 +1,789 @@
+        libspdm_fill_measurement_hem_block (spdm_hem, m_libspdm_use_measurement_hash_algo)) {
+        assert_true(false);
+    }
+
+    status = libspdm_get_measurement_extension_log(spdm_context, NULL, spdm_hem,
+                                                   (uint32_t)spdm_hem_len, &spdm_mel_size,
+                                                   spdm_mel);
+    assert_int_equal(status, LIBSPDM_STATUS_SEND_FAIL);
+
+    free(spdm_hem);
+}
+
+/**
+ * Test 2: Normal case, request a certificate chain
+ * Expected Behavior: receives a valid certificate chain with the correct number of Certificate messages
+ **/
+void libspdm_test_requester_get_measurement_extension_log_case2(void **state)
+{
+    libspdm_return_t status;
+    libspdm_test_context_t *spdm_test_context;
+    libspdm_context_t *spdm_context;
+    size_t spdm_mel_size;
+    uint8_t spdm_mel[LIBSPDM_MAX_CERT_CHAIN_SIZE];
+    size_t spdm_hem_len;
+    void *spdm_hem;
+
+    spdm_test_context = *state;
+    spdm_context = spdm_test_context->spdm_context;
+    spdm_test_context->case_id = 0x2;
+    spdm_context->connection_info.version = SPDM_MESSAGE_VERSION_13 <<
+                                            SPDM_VERSION_NUMBER_SHIFT_BIT;
+    spdm_context->connection_info.connection_state =
+        LIBSPDM_CONNECTION_STATE_AUTHENTICATED;
+    spdm_context->connection_info.capability.flags |=
+        SPDM_GET_CAPABILITIES_RESPONSE_FLAGS_MEL_CAP;
+
+    libspdm_reset_message_b(spdm_context);
+    spdm_mel_size = sizeof(spdm_mel);
+    libspdm_zero_mem(spdm_mel, sizeof(spdm_mel));
+
+    spdm_context->connection_info.algorithm.measurement_spec =
+        m_libspdm_use_measurement_spec;
+    spdm_context->connection_info.algorithm.measurement_hash_algo =
+        m_libspdm_use_measurement_hash_algo;
+    spdm_context->connection_info.algorithm.base_hash_algo =
+        m_libspdm_use_hash_algo;
+    spdm_context->connection_info.algorithm.base_asym_algo =
+        m_libspdm_use_asym_algo;
+    spdm_context->local_context.algorithm.measurement_spec =
+        SPDM_MEASUREMENT_SPECIFICATION_DMTF;
+
+    spdm_hem_len = libspdm_fill_measurement_hem_block (NULL, m_libspdm_use_measurement_hash_algo);
+
+    spdm_hem = malloc(spdm_hem_len);
+    if (spdm_hem == NULL) {
+        assert_true(false);
+    }
+
+    if (spdm_hem_len !=
+        libspdm_fill_measurement_hem_block (spdm_hem, m_libspdm_use_measurement_hash_algo)) {
+        assert_true(false);
+    }
+
+    status = libspdm_get_measurement_extension_log(spdm_context, NULL, spdm_hem,
+                                                   (uint32_t)spdm_hem_len, &spdm_mel_size,
+                                                   spdm_mel);
+    assert_int_equal(status, LIBSPDM_STATUS_SUCCESS);
+    free(spdm_hem);
+}
+
+/**
+ * Test 3: Under normal circumstances, request the certificate chain with the largest size
+ * Expected Behavior: receives a valid certificate chain with the correct number of Certificate messages
+ **/
```

Comment:
`certificate chain` comment is not right.
it should be `measurement_extension_log`, not `certificate chain`.